README.md

```diff
@@ -20,11 +49,65 @@ The DICOM to NifTi conversion in the package is based on a wrapper around the [d
 
 ## NifTi to DICOM-SEG
 
+The NifTi to DICOM-SEG within nekton converts incoming segmentation NifTi to DICOM-SEG. The matching of the segmentation index to a text label is 
+done via json file using the schema suggested by `dcmqi`. The json can be generated using the [gui](http://qiicr.org/dcmqi/#/seg) also an example can be seen [here](https://github.com/deepc-health/nekton/blob/master/tests/test_data/sample_segmentation/mapping.json). 
+
+Currently, `nekton` supports creation of multiclass DICOM-SEG of two types-
+
+- single layer DICOM-SEG, where each non-empty slice has an individual file
+- multi layer DICOM-SEG, where all the n slices are rolled into a single file
+
 ### Usage
 
-TBD
+1. NifTi to single layer DICOM-SEG
 
-## NifTi to GSPS
+```python
+from nekton.nii2dcm import Nii2DcmSeg
+import glob
+converter = Nii2DcmSeg()
+path_dcms = [path for path in glob.glob(dir_dcms)]
+path_mapping = "mapping.json"
+path_seg_nifti = "CT5N_segmentation.nii.gz"
+dcmsegs = converter_dcmseg.multiclass_converter(
+        segfile = path_seg_nifti, segMapping= path_mapping, dcmfiles =path_dcms, multiLayer=False
+    )
+print (len(dcmsegs))
+# 3
+```
+
+2. NifTi to multi layer DICOM-SEG
+
+```python
+from nekton.nii2dcm import Nii2DcmSeg
+import glob
+converter = Nii2DcmSeg()
+path_dcms = [path for path in glob.glob(dir_dcms)]
```

Comment:
This could have been done within the `multiclass_converter` method directly. The method could take and in_dir and out_dir, parse the in_dir and put everything in the out_dir

Reply:
This was done on purpose. We do not want to deal with different series, form different windowing, etc, within the `multiclass_converter` and want the user to provide correct matching dicoms.